send-failure-to-sentry.js

```diff
@@ -0,0 +1,8 @@
+const { env } = require('node:process');
+const Sentry = require('@sentry/node');
+
+Sentry.init({
+  dsn: env.SENTRY_DSN,
+});
+
+Sentry.captureMessage("Something went wrong.");
```

Comment:
This is ridiculously simple for now, but at least we have an alert. Will open a follow-on issue to give this some more context.